src/test/java/core/FreestyleJobTest.java

```diff
@@ -94,13 +96,14 @@ public void should_visit_build_with_permalink() {
         Build build = job.scheduleBuild().shouldSucceed();
         job.open();
         WebElement link = job.find(By.partialLinkText("Last build (#1)"));
-        link.click();
-
-        assertThat(driver, hasContent("Build #1"));
-        assertThat(driver, hasContent("No changes"));
-
-        WebElement successIcon = build.find(By.xpath("//h1/img"));
-        assertThat(successIcon.getAttribute("tooltip"), is("Success"));
+        String expectedUrl = link.getAttribute("href");
```

Comment:
I thought about switching this to match the SVG's title but a quick attempt did not work (and it seems that the SVN has some asynchronous javascript associated with it to manipulate the DOM).

As the `Build` `PageObject` already has a permalink option (it does not use the permalink.click though) use that and as the page URL is hard coded check that the permalink link was actually reffering to this page (incase the Permalink regresses in the future)